eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/ClusteredVariantsRestControllerTest.java

```diff
@@ -561,10 +558,62 @@ public void testGetMergedAndDeprecatedVariants()
         ResponseEntity<List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>>> response =
                 testRestTemplate.exchange(getVariantUrl, HttpMethod.GET, null, new ClusteredVariantType());
 
+        // then
+        assertEquals(HttpStatus.GONE, response.getStatusCode());
+        assertEquals(1, response.getBody().size());
+        assertEquals(clusteredVariantEntity3.getModel(), response.getBody().get(0).getData());
+        assertClusteredVariantCreatedDateNotNull(response.getBody());
+    }
+
+    @Test
+    public void testGetDeprecatedDbsnpClusteredVariant()
+            throws AccessionCouldNotBeGeneratedException, AccessionMergedException, AccessionDoesNotExistException,
+                   AccessionDeprecatedException {
+        // given
+        clusteredService.deprecate(DBSNP_CLUSTERED_VARIANT_ACCESSION_1, "deprecated for testing");
+        String getVariantUrl = URL + DBSNP_CLUSTERED_VARIANT_ACCESSION_1;
+
+        // when
+        ResponseEntity<List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>>> response =
+                testRestTemplate.exchange(getVariantUrl, HttpMethod.GET, null, new ClusteredVariantType());
+
         // then
         assertEquals(HttpStatus.GONE, response.getStatusCode());
         assertEquals(1, response.getBody().size());
         assertEquals(clusteredVariantEntity1.getModel(), response.getBody().get(0).getData());
         assertClusteredVariantCreatedDateNotNull(response.getBody());
```

Comment:
Could you please add a new test or modify the existing test to test return values in the case of multiple inactive objects.

Reply:
I will change it so that there are several *documents* with the same accession. several inactive objects won't happen